host/traceh.c

```diff
@@ -224,7 +209,31 @@ void oe_log_message(bool is_enclave, oe_log_level_t level, const char* message)
                 _log_creation_failed_before = true;
                 return;
             }
-            _write_message_to_stream(log_file, is_enclave, level, message);
+
+            char* message_dup = strdup(message);
+            char* log_msg = strtok(message_dup, "[");
+            size_t len = strlen(log_msg);
+            if (log_msg[len - 1] == '\n')
+            {
+                log_msg[len - 1] = 0;
+            }
+            char* file_name = strtok(NULL, ":");
+            char* function = strtok(NULL, ":");
+            char* line_number = strtok(NULL, "]");
+
+            _write_custom_format_message_to_stream(
```

Comment:
Can we avoid parsing the message and instead create the message using the correct format in the first place?
As written currently, the log that gets printed in stdout has a different format from the log that gets printed in a file.

Reply:
Yes thank you, I have changed it to log the message into the file in the same format as the default formatter if the custom one is missing.